File: Java/Ayudantías/src/A10/Nodo.java
package A10;

public class Nodo {
	private Nodo siguiente;
	private Jugador jugador;
	public Nodo(Jugador jugador) {
		this.jugador = jugador;
	}
	public Nodo getSiguiente() {
		return siguiente;
	}
	public void setSiguiente(Nodo siguiente) {
		this.siguiente = siguiente;
	}
	public Jugador getJugador() {
		return jugador;
	}
}

File: C++/Pilas y  filas/Nodo.cpp
#include <iostream>
using namespace std;

class Nodo{
    private:
        double dato;
        Nodo* siguiente;
    public:
        Nodo(){
            dato = 0;
            siguiente = NULL;
        };
        double getDato(){
            return dato;
        };
        Nodo* getSiguiente(){
            return siguiente;
        };
        void setDato(double _dato){
            dato = _dato;
        };
        void setSiguiente(Nodo* _siguiente){
            siguiente = _siguiente;
        };
};